sub-packages/bionemo-esm2/src/bionemo/esm2/scripts/finetune_esm2.py

```diff
@@ -362,7 +390,7 @@ def train_model(
         autocast_dtype=get_autocast_dtype(precision),  # setting this speeds things up a lot
         tensor_model_parallel_size=tensor_model_parallel_size,
         pipeline_model_parallel_size=pipeline_model_parallel_size,
-        initial_ckpt_path=str(restore_from_checkpoint_path),
+        initial_ckpt_path=str(restore_from_checkpoint_path) if restore_from_checkpoint_path is not None else None,
```

Comment:
Wait... what does fine-tuning mean if you do not have a `restore_from_checkpoint_path`? If this change was just for typing, better would be to make `restore_from_checkpoint_path` required... Please add a code comment on when a user would want to not have a checkpoint supplied during fine-tuning.